examples/sudt-script/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+resolver = "2"
+
+members = [
+  # Please don't remove the following line, we use it to automatically
```

Comment:
@linnnsss  I am going to ignore all the comment text suggestion in the `examples/sudt-script` project since it is generated automatically by [ckb-script-templates](https://github.com/cryptape/ckb-script-templates), maybe it is better to submit these suggestion  directly to ckb-script-templates project.